assignments/prototypes.js

```diff
@@ -23,6 +32,18 @@
   * should inherit destroy() from GameObject's prototype
 */
 
+function CharacterStats(hp, name) {
+  this.hp = hp;
+  this.name = name;
+  GameObject.call(this, createdAt, dimensions)
+}
+
+CharacterStats.prototype = function () {
+  `${this} took damage.`;
+}
+
+CharacterStats.prototype = Object.create(GameObject.prototype);
```

Comment:
Make sure to create the prototype *before* adding prototypes to CharacterStats so it doesn't override it.